packages/drivers/routerlicious-driver/src/urlUtils.ts

```diff
@@ -17,3 +19,25 @@ export const parseFluidUrl = (fluidUrl: string): URLParse => {
  */
 export const replaceDocumentIdInPath = (urlPath: string, documentId: string): string =>
     urlPath.split("/").slice(0, -1).concat([documentId]).join("/");
+
+export const createFluidUrl = (domain: string, pathname: string): string =>
+    "fluid://".concat(domain, pathname);
+
+export const replaceWithDiscoveredUrl = (resolvedUrl: IFluidResolvedUrl,
+    session: ISession,
+    parsedUrl: URLParse): void => {
+    if (session && session.ordererUrl.includes("https")) {
+        resolvedUrl.url = createFluidUrl(session.ordererUrl.replace(/^https?:\/\//, ""), parsedUrl.pathname);
```

Comment:
As previously commented by @pradeepvairamani, moving `discoveredOrdererUrl` above this and using `discoveredOrdererUrl.hostname` instead of a replace RegEx would be better

Reply:
Got it. Thanks. Done by use discoveredOrdererUrl.host